tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-initial-dialog.html

```diff
@@ -28,24 +28,47 @@
     <paper-dialog id="dialog"
                   modal
                   opened="{{_open}}"
-                  width="320"
                   with-backdrop="[[_useNativeBackdrop]]">
-      <h2 id='dialog-title'>Waiting for first Session.run() to connect...</h2>
+      <h2 id='dialog-title'>Debugger is waiting for Session.run() connections...</h2>
       <div class='code-example'>
-        <!-- TODO(cais): Rename id. -->
-        <pre id="session-run-code-example">
-# To connect to the debugger from your tf.Session:
+        <div class="code-example-section">
+          <div class="code-example-section-title">
+            <a href="https://www.tensorflow.org/api_docs/python/tf/Session" target="_blank">tf.Session</a>:
+          </div>
+          <pre class="code-snippet">
 from tensorflow.python import debug as tf_debug
 sess = tf.Session()
 sess = tf_debug.TensorBoardDebugWrapperSession(sess, "[[_host]]:[[_port]]")
 sess.run(my_fetches)
         </pre>
-        <pre id="hook-code-example">
-# To connect to the debugger using hooks, e.g., from tf.Estimator:
+        </div>
+        <div class="code-example-section">
+          <div class="code-example-section-title">
+            <a href="https://www.tensorflow.org/programmers_guide/estimators" target="_blank">Estimator</a>
+            |
+            <a href="https://www.tensorflow.org/api_docs/python/tf/train/MonitoredSession" target="_blank">MonitoredSession</a>:
+          </div>
+          <pre class="code-snippet">
 from tensorflow.python import debug as tf_debug
 hook = tf_debug.TensorBoardDebugHook("[[_host]]:[[_port]]")
 my_estimator.fit(x=x_data, y=y_data, steps=1000, monitors=[hook])
-        </pre>
+          </pre>
+        </div>
+        <div class="code-example-section">
+          <div class="code-example-section-title">
+            <a href="https://keras.io/models/model/" target="_blank">Keras Model</a>:
+          </div>
+          <pre class="code-snippet">
+import tensorflow as tf
```

Comment:
nit: this is still inconsistent with the tf.Session() example

Reply:
Good catch. Fixed.